README.md

```diff
@@ -1,2 +1,51 @@
 # playbook
-collection of venturemark playbooks
+
+Collection of playbooks for the Venturemark platform.
+
+
+
+### What Is A Playbook?
+
+A playbook aims to provide you with inspiration and instruction for how to use
+the Venturemark platform. The Venturemark platform provides you with an abstract
+and powerful way to combine contextual information and factual data. Due to the
+abstract nature of the Venturemark platform, you can use it in many different
+ways. There are many powerful use cases for high performing individuals to use
+the Venturemark platform. A playbook is merely a piece of documentation
+providing you with instructions for habit forming processes. These processes aim
+to achieve a track record of better results throughout your venture's journey.
+
+
+
+### How To Use A Playbook?
+
+A playbook should be used in a way that is most suitable for you and your
+venture. In short, you decide how to use a playbook. In some cases you are
+better off following a playbook strictly. In some cases you are better off
+following a playbook rather loosely. You decide how suitable a playbook is for
+you and your venture. If you feel a playbook does not serve you well then stop
+using it. Try to find the right playbooks that help you to be better tomorrow
+and let your track record speak for itself.
+
+
+
+### How To Find The Right Playbook For Me?
+
+All of our playbooks are documented in this repository. For now you can apply
+the shotgun method and try out a hand full of playbooks that you find
+interesting. You can then refine and adjust the usage of the playbooks that
+serve you and your venture in the best possible way.
+
+
+
+### How To Create My Own Playbook?
+
```

Comment:
Fine for this as placeholder text - but it does not say how to create a playbook and is too long.

Reply:
That does not help to move forward. How to rewrite it? I agree it is too long, though I am not sure what you are missing and how to write it better now. It would be cool to provide alternatives with your critique.

Reply:
This is fine as for now.

My comment is that I didn't get any increase in understanding how to create a playbook. Marketing language would have a general overview and then this is not specific enough for step by step; i.e. 1. create a timeline 2. create a post, etc.